packages/node-terra/src/indexer/apiterra.service.ts

```diff
@@ -233,3 +250,65 @@ export class TerraClient {
     return this.baseApi;
   }
 }
+
+export class TerraSafeApi implements ITerraSafeApi {
+  preferredConnection: AxiosInstance;
+  wasm: SafeWasmApi;
+  constructor(
+    private _lcdConnection: AxiosInstance,
+    private height: number,
+    mantlmintHealthOK: boolean,
+    private _mantlemintConnection?: AxiosInstance,
+  ) {
+    this.preferredConnection = mantlmintHealthOK
+      ? this._mantlemintConnection
+      : this._lcdConnection;
+    this.wasm = new SafeWasmApi(this.preferredConnection, this.height);
+  }
+}
+
+class SafeWasmApi implements ISafeWasmApi {
+  constructor(
+    private preferredConnection: AxiosInstance,
+    private height: number,
+  ) {}
+  async codeInfo(codeID: number): Promise<CodeInfo> {
+    const { data } = await this.preferredConnection.get(
+      `/terra/wasm/v1beta1/codes/${codeID}`,
+    );
+    return <CodeInfo>{
+      code_id: Number.parseInt(data.code_info.code_id),
+      code_hash: data.code_info.code_hash,
+      creator: data.code_info.creator,
+    };
+  }
+
+  async contractInfo(contractAddress: string): Promise<ContractInfo> {
```

Comment:
Do these 2 methods support specifying the height parameter?

Reply:
No, looks like there is no height parameter on these methods.